render.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [ $# -ne 2 ]
+then 
+    echo requires two arguments
+    exit 1
+fi
+
+placeholder=$1
+versionfile=$2
+
+render() {
+    sub="s~$placeholder~$2~g;"
```

Comment:
can we name $1 and $2 here so it's less confusing? Also should this be renderline so its more clear you're not calling your script recurisively?